examples/ReorderExample.js

```diff
@@ -44,7 +28,7 @@ class ReorderExample extends React.Component {
     super(props);
 
     this.state = {
-      dataList: new FakeObjectDataListStore(1000000),
+      dataList: new FakeObjectDataListStore(10000),
```

Comment:
Ya already using production react, but will switch this number back.  I had just noticed the jankiness while dev'ing